CHANGELOG.next.asciidoc

```diff
@@ -110,6 +110,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Fix panic in TCP and UDP inputs on Linux when collecting socket metrics from OS. {issue}35064[35064]
 - Correctly collect TCP and UDP metrics for unspecified address values. {pull}35111[35111]
 - Fix base for UDP and TCP queue metrics and UDP drops metric. {pull}35123[35123]
+- Sanitize filenames for request tracer in httpjson and cel inputs. {pull}35143[35143]
```

Comment:
There are no changes to CEL in this PR. Is that a mistake in the changelog or did you forget to add something? I'm pretty sure we need the same changes in CEL.

Reply:
Added fixes in https://github.com/elastic/beats/pull/35154/files